build.psm1

```diff
@@ -551,30 +551,32 @@ function Start-PSPester {
     [CmdletBinding()]param(
         [string]$OutputFormat = "NUnitXml",
         [string]$OutputFile = "pester-tests.xml",
-        [switch]$DisableExit,
         [string[]]$ExcludeTag = "Slow",
         [string[]]$Tag = "CI",
-        [string]$Path = "$PSScriptRoot/test/powershell"
+        [string]$Path = "$PSScriptRoot/test/powershell",
+        [switch]$ThrowOnFailure,
+        [switch]$FullCLR,
+        [string]$binDir = (Split-Path (New-PSOptions -FullCLR:$FullCLR).Output) 
     )
 
-    $powershell = Get-PSOutput
-
     # All concatenated commands/arguments are suffixed with the delimiter (space)
     $Command = ""
+    $powershell = Join-Path $binDir 'powershell'
 
     # Windows needs the execution policy adjusted
     if ($IsWindows) {
         $Command += "Set-ExecutionPolicy -Scope Process Unrestricted; "
```

Comment:
I'm not sure if this is needed any more... unrestricted is the default... I just didn't want to change it.